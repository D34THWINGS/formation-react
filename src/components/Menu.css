.menu {
  display: flex;
  align-items: center;
  position: fixed;
  top: 0;
  left: 0;
  width: 100%;
  height: 4rem;
  padding: 0 1rem;
  background: #333;
  text-align: left;
}

.menu__title {
  margin: 0;
  color: white;
}
