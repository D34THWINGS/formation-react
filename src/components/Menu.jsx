import React from 'react';
import styled from '@emotion/styled';

const StyledMenu = styled.header`
  display: flex;
  align-items: center;
  position: fixed;
  top: 0;
  left: 0;
  width: 100%;
  height: 4rem;
  padding: 0 1rem;
  background: #333;
  text-align: left;
`;

const MenuTitle = styled.h1`
  margin: 0;
  color: white;
`;

const Menu = () => (
  <StyledMenu>
    <MenuTitle>Choixpeau</MenuTitle>
  </StyledMenu>
);

export default Menu;
