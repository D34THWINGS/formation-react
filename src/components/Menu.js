import React from 'react';
import './Menu.css';

const Menu = () => (
  <header className="menu">
    <h1 className="menu__title">Choixpeau</h1>
  </header>
);

export default Menu;
