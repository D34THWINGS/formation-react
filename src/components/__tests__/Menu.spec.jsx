import React from 'react';
import { shallow } from 'enzyme';

import Menu from '../Menu';

describe('Menu', () => {
  it('should render app menu', () => {
    const wrapper = shallow(<Menu />);
    expect(wrapper).toMatchSnapshot();
  });
});
