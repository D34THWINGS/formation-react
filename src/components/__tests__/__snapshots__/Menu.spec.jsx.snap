// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Menu should render app menu 1`] = `
<header
  className="menu"
>
  <h1
    className="menu__title"
  >
    Choixpeau
  </h1>
</header>
`;
